platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -0,0 +1,648 @@
+/* Copyright (c) 2021, ARM Limited
```

Comment:
This file is mostly an exact copy of odp_crypto_null.c, which says "Copyright (c) 2014-2018, Linaro Limited". It is not appropriate to remove the copyright attribution.

Reply:
Okay, we will add the other copyright too in addition to this one. Thanks!